### ISSUE TITLE
Two RectangleLabels don't work together.

### ISSUE THREAD

**Issue by samirbr** [heartexlabs/label-studio#202]
**Describe the bug**
I have a case where the same image can be labeled for different datasets. I added two RectangleLabels. One for common personal objects (purse, backpack, etc), other for people. 
The UI render both controls, but only the latest work. I click on `Person` and I can draw a bbox, but if I click `Purse`, or `Backpack` nothing happens.

**To Reproduce**
1. Go to https://labelstud.io/playground/
2. Paste in LabelConfig.
```
<View>
  <Image name="img" value="$image"/>
  
  <RectangleLabels name="common" toName="img">
   <Label value="Purse" background="green"/>
   <Label value="Backpack" background="blue"/>
  </RectangleLabels>
  
  <RectangleLabels name="person" toName="img">
   <Label value="Person" background="black"/>
  </RectangleLabels>
</View>
```
3. Add a bbox for Person. It works!
4. Add a bbox for Backpack. Nothing happens.

**Expected behavior**
Add a bbox for Backpack. It works!

**Screenshots**
![Screenshot from 2020-01-27 16-01-08](https://user-images.githubusercontent.com/1590543/73224422-4cb55d00-411e-11ea-9f65-85a29a615148.png)

**Environment (please complete the following information):**
 - OS: Linux
 - Browser Chrome
 - Version 79

**Comment by deppp** [comment]
@samirbr thanks for the report. I can reproduce, will be fixing it later this week.

**Comment by makseq** [comment]
This bug was fixed in [release/1.1.0](https://github.com/heartexlabs/label-studio/tree/release/1.1.0). Feel free to reopen if you have this problem again.
